packages/form/test/lion-form.test.js

```diff
@@ -138,6 +138,23 @@ describe('<lion-form>', () => {
     expect(submitEv.composed).to.be.false;
   });
 
+  it('redispatches a submit event on the native form node when calling submit() imperatively', async () => {
+    const nativeFormSubmitEventSpy = spy();
+    const el = await fixture(html`
+      <lion-form>
+        <form @submit=${nativeFormSubmitEventSpy}>
+          <button type="submit">submit</button>
+        </form>
+      </lion-form>
+    `);
+    const submitSpy = spy(el, 'submit');
+    const submitGroupSpy = spy(el, 'submitGroup');
+    el.submit();
+    expect(submitSpy.calledOnce).to.be.true;
+    expect(nativeFormSubmitEventSpy.calledOnce).to.be.true;
```

Comment:
Maybe also test here that this event is cancellable now?
(for a future me to understand what this was about)